server/models/db/appointment.js

```diff
@@ -0,0 +1,16 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+
+const Tables = require('../../constants/tables.json');
+const AppointmentStatus = require('../../constants/appointment-status.json');
+
+// This method inserts an appointment entry given specific information.
+async function insertAppointment(studentId, workerTimeslotId, purpose, studentNotes, workerComments) {
```

Comment:
I separated the actual database manipulation query to comply with the desired backend flow from routes > handlers > db.

Reply:
thanks for specifying!